modal/_utils/logger.py

```diff
@@ -4,35 +4,45 @@
 
 
 def configure_logger(logger: logging.Logger, log_level: str, log_format: str):
+    from modal.config import config
+
     ch = logging.StreamHandler()
     log_level_numeric = logging.getLevelName(log_level.upper())
     logger.setLevel(log_level_numeric)
     ch.setLevel(log_level_numeric)
-
+    datefmt = "%Y-%m-%dT%H:%M:%S%z"
     if log_format.upper() == "JSON":
         # This is primarily for modal internal use.
         # pythonjsonlogger is already installed in the environment.
         from pythonjsonlogger import jsonlogger
 
-        json_formatter = jsonlogger.JsonFormatter(
-            fmt=(
+        if not (log_format_pattern := config.get("log_pattern")):
+            log_format_pattern = (
```

Comment:
Would users be expected to specify something like this in their `.modal.toml`?

Reply:
Typically no, but they could, similar to how they could set log_format="JSON" there today

Reply:
I mostly added it as a configurable fallback option in case there are users relying on the current logging format for log ingestion or similar